lep/DRAFT/011_Performance_Monitor_Goals_and_Metrics.md

```diff
@@ -0,0 +1,85 @@
+#LEP 011 - Performance Monitor Goals and Metrics
+
+Date:   March 24, 2015
+
+Status: Draft
+
+Author: fffw
+
+## Abstract
+
+To continuously monitor the performance of Lantern network, we need to define dimensions and metrics clearly, and collect data to fulfill the goal at both given and get mode.
+
+We must also keep user's anonymity.
+
+## Proposal
+
+The output, is a multi-dimensional model which can pivot from any dimension. The presentation solution is TBD, hopefully there are existing services or open source projects.
+The fact to measure is data flow. For example, we can query:
+* Average time required to establish connection from Iran to Facebook last weekend
```

Comment:
Sorry for the inappropriate words. It's user perceptional delay to access Facebook page.